docs/src/storage.md

```diff
@@ -51,65 +51,65 @@ we recommend that you also consider this aspect when you choose your storage
 solution, especially if you manage very large databases.
 
 !!! Important
-    Please refer to the official Kubernetes documentation for a list of all
-    the supported [Container Storage Interface (CSI) drivers](https://kubernetes-csi.github.io/docs/drivers.html)
-    that provide snapshotting capabilities.
+    See the Kubernetes documentation for a list of all
+    the supported [container storage interface (CSI) drivers](https://kubernetes-csi.github.io/docs/drivers.html)
+    that provide snapshot capabilities.
 
 ## Benchmarking CloudNativePG
 
-We recommend that you benchmark CloudNativePG in a controlled Kubernetes
-environment, before deploying the database in production, by following
-the [guidelines in the "Benchmarking" section](benchmarking.md).
+Before deploying the database in production, we recommend that you benchmark 
+CloudNativePG in a controlled Kubernetes environment. Follow
+the guidelines in [Benchmarking](benchmarking.md).
 
-Briefly, our advice is to operate at two levels:
+Briefly, we recommend operating at two levels:
 
-- measuring the performance of the underlying storage using `fio`, with relevant
+- Measuring the performance of the underlying storage using fio, with relevant
   metrics for database workloads such as throughput for sequential reads, sequential
-  writes, random reads and random writes
-- measuring the performance of the database using the default benchmarking tool
-  distributed along with PostgreSQL: `pgbench`
+  writes, random reads, and random writes
```

Comment:
In this case, it's a list of items, none of which are sentences. 

If it were a definition list (which also might have phrases and not sentences), I try to always use this form:

**Thing being defined** em-dash The definition, ending with a period.

I am not 100% sure all of this thinking is captured in the style guide. I will locate the style guide online and send a link, but this might have to wait until the new year.